test/fixtures/kitchen/tsconfig.json

```diff
@@ -2,13 +2,11 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "build"
+    "outDir": "build",
+    "esModuleInterop": false // test default setting
```

Comment:
Why is this needed for tests?

Reply:
This ensures I won't break it again :) This is run as part of the kitchen sink.  If I used `import http from 'http'` now, this test would fail.